app/src/main/java/org/mifos/mobile/ui/fragments/ClientAccountsFragment.kt

```diff
@@ -12,15 +12,11 @@ import android.widget.Toast
 import androidx.appcompat.widget.SearchView
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
-import androidx.viewpager.widget.ViewPager
 import androidx.viewpager.widget.ViewPager.OnPageChangeListener
-import butterknife.BindView
-import butterknife.ButterKnife
-import butterknife.OnClick
 import com.google.android.material.dialog.MaterialAlertDialogBuilder
-import com.google.android.material.floatingactionbutton.FloatingActionButton
 import com.google.android.material.tabs.TabLayout
```

Comment:
do we need TabLayout ? What I make of it is that we are removing its usage from line 51 . Do cross check and remove it if its unused

Reply:
Yes we are removing usage of it. 
But it's still being automatically imported by the IDE because there's a reference to it in the same file using a javadoc comment. If I'll remove the import then this javadoc comment will no more reference us when we hover over it. I don't know what to make of it(🤔). Let's leave it there. 

<img width="639" alt="Screenshot 2023-05-25 at 19 37 32" src="https://github.com/openMF/mifos-mobile/assets/59947871/df662274-991d-4a3f-a804-b271338ddaa5">

On Hovering ->
<img width="464" alt="Screenshot 2023-05-25 at 19 39 06" src="https://github.com/openMF/mifos-mobile/assets/59947871/1fb7934a-3ebe-4973-82d9-4f20d6d0a82c">